src/components/learner-credit-management/cards/CourseCardFooterActions.jsx

```diff
@@ -27,32 +26,34 @@ const CourseCardFooterActions = ({ enterpriseId, course }) => {
       { courseUuid: uuid },
     );
   };
-  return [
-    <Button
-      key="link-to-course"
-      as={Hyperlink}
-      data-testid="hyperlink-view-course"
-      onClick={handleViewCourse}
-      destination={linkToCourse}
-      target="_blank"
-      variant="outline-primary"
-    >
-      <FormattedMessage
-        id="lcm.budget.detail.page.catalog.tab.course.card.view.course"
-        defaultMessage="View course"
-        description="Button text to view course"
-      />
-    </Button>,
-    (!catalogGroupView ? (
-      <NewAssignmentModalButton key="assignment-modal-trigger" course={course}>
+  return (
+    <>
+      <Button
+        key="link-to-course"
```

Comment:
nit: technically i believe the `key` props here and below could be removed as these components are no longer rendered within an array like before.